packages/kbn-observable/src/operators/mergeMap.ts

```diff
@@ -101,26 +66,9 @@ export function mergeMap<T, I, R>(
           }
           active++;
 
-          let innerIndex = 0;
-
           $from(result).subscribe({
             next(innerValue) {
-              let result: R | I = innerValue;
-
-              if (resultSelector !== undefined) {
-                try {
-                  result = resultSelector(
-                    value,
-                    innerValue,
-                    outerIndex,
-                    innerIndex
-                  );
-                } catch (err) {
-                  destination.error(err);
-                  return;
-                }
-                innerIndex++;
-              }
+              let result: R = innerValue;
```

Comment:
__nit:__ why not just `destination.next(innerValue);`?

Reply:
:+1: